tools/shared/src/main/scala/org/scalajs/core/tools/optimizer/OptimizerCore.scala

```diff
@@ -2168,6 +2168,18 @@ private[optimizer] abstract class OptimizerCore(
     }
   }
 
+  private def literalToString(l: Literal): String = l match {
+    case StringLiteral(str) => str
+    case FloatLiteral(flt) => flt.toDouble.toString
+    case DoubleLiteral(dbl) if dbl.isValidInt => dbl.toInt.toString
+    case DoubleLiteral(dbl) => dbl.toString
+    case LongLiteral(lng) => lng.toString
+    case IntLiteral(intgr) => intgr.toString
+    case BooleanLiteral(bln) => bln.toString
+    case Null() => "null"
+    case Undefined() => "undefined"
```

Comment:
Align the arrows. See the coding style at https://github.com/scala-js/scala-js/blob/master/CODINGSTYLE.md#pattern-matching